### PR TITLE
[record-minmax] Fix boundary percentile cases

### PR DESCRIPTION
**Pull request by binarman**
This commit fixes computation of percentile 0 and 100.

ONE-DCO-1.0-Signed-off-by: Alexander Efimov <a.efimov@samsung.com>

### CODE REVIEW

File: compiler/record-minmax/include/RecordFunction.h [hunk 1]
```diff
@@ -33,12 +33,6 @@ float getNthPercentile(std::vector<float> &vector, float percentile)
   if (percentile < 0 || percentile > 100)
     throw std::runtime_error("Percentile must be ranged from 0 to 100");
 
-  if (percentile == 0.0)
-    return vector.front();
```
Comment (binarman) [inline compiler/record-minmax/include/RecordFunction.h:37]: The problem here: vector is not sorted at this point.

**Review by YiHyunJin — APPROVED** [review]
LGTM

File: compiler/record-minmax/include/RecordFunction.h (hunk 1 above)
Comment (jinevening) [reply compiler/record-minmax/include/RecordFunction.h:37]: Wow..

**Review by jinevening — APPROVED** [review]
LGTM Thank you

**Pull request merged**